example/index.ts

```diff
@@ -1,55 +1,37 @@
-import { rAF, forceV, applyForce, vector as Vector, Entity } from "../src";
-
-interface UseGravityArgs {
-  moverMass: number;
-  attractorMass: number;
-  attractor: Vector<number>;
-}
-
-const applyForceForStep = (config: UseGravityArgs, mover: Entity) => {
-  const force = forceV({
-    ...config,
-    mover: mover.position
-  });
-
-  return applyForce({
-    force,
-    entity: {
-      mass: config.moverMass,
-      acceleration: mover.acceleration,
-      velocity: mover.velocity,
-      position: mover.position
+import { gravity, vector } from "../src";
+
+const el = document.createElement("div");
+document.body.appendChild(el);
+
+const positionAttractor = [500, 500] as vector<number>;
+const attractorEl = document.createElement("div");
+document.body.appendChild(attractorEl);
+
+el.style.height = "10px";
+el.style.width = "10px";
+el.style.borderRadius = "50%";
+el.style.background = "steelblue";
+
+attractorEl.style.height = "25px";
+attractorEl.style.width = "25px";
+attractorEl.style.borderRadius = "50%";
+attractorEl.style.background = "orange";
+attractorEl.style.left = `${positionAttractor[0]}px`;
+attractorEl.style.top = `${positionAttractor[1]}px`;
+attractorEl.style.position = "absolute";
+
+const [stop] = gravity({
+  config: {
+    moverMass: 2000,
+    attractorMass: 100000000,
+    attractor: positionAttractor,
+    initialVelocity: [0.01, 0.05],
+    threshold: {
+      min: 10,
+      max: 750
     }
-  });
-};
-
-const run = (config: UseGravityArgs) => {
-  const { start, stop } = rAF();
-
-  start((timestamp, lastFrame, mover) => {
-    const diffTime = timestamp > lastFrame + 64 ? 0 : lastFrame;
-    const steps = Math.floor(timestamp - diffTime);
-
-    let entity: Entity;
-
-    if (steps > 0) {
-      for (let i = 0; i < steps; i++) {
-        entity = applyForceForStep(config, mover);
-      }
-    } else {
-      entity = applyForceForStep(config, mover);
-    }
-
-    return entity;
-  });
-
-  setTimeout(() => {
-    stop();
-  }, 1000);
-};
-
-run({
-  moverMass: 20,
-  attractorMass: 10000,
-  attractor: [500, 500]
+  },
+  onFrame: ([x, y]) => {
+    el.style.transform = `translate(${x}px, ${y}px)`;
+  }
```

Comment:
This is where I figure we can apply interpolations to convert `position` to normalized values. For example, applying a `lerp` here might be really useful to get 0 - 1 values for properties like `opacity`. We may need to pass back more state for that to happen, however. I'm still not sold on whether or not we leave this up entirely to the user like `react-pose` does with "passive" values: https://popmotion.io/pose/learn/passive/